docs/mkdocs.yml

```diff
@@ -84,14 +84,16 @@ nav:
             -   Containers: ci/Containers.md
             -   Logging: ci/Logging.md
     -   Testing:
-            -   Test Runner: test/Runner.md
             -   Flaky annotation: test/FlakyAnnotation.md
             -   Toast: test/Toast.md
     -   Test framework:
             - test_framework/TestFramework.md
             -   Snackbar: test_framework/Snackbar.md
             -   Screen checks: test_framework/ScreenChecks.md
             -   Internals: test_framework/Internals.md
+    -   Test runner:
```

Comment:
How about the `projects` page?
We used it for ready to use plugins

Reply:
I know, but it's not ready to use for now